usr/share/rear/conf/GNU/Linux.conf

```diff
@@ -184,6 +184,7 @@ env
 w
 dosfslabel
 sysctl
+blockdev
```

Comment:
@rmetrich 
please also add the 'lsblk' to be included in the recovery system
because at least for me it is very helpful to be able to call 'lsblk'
also in the recovery system.